tailwind.config.ts

```diff
@@ -52,6 +52,14 @@ const config = {
           DEFAULT: "hsl(var(--card))",
           foreground: "hsl(var(--card-foreground))",
         },
+        brand: {
+          DEFAULT: "hsl(var(--brand-pink))",
+          light: "hsl(var(--brand-light-pink))",
+          white: "hsl(var(--brand-white))",
+        },
```

Comment:
Just for my education, is this what makes `bg-brand`, `bg-brand-light` work?

Reply:
Yes, that is correct. I copied the current way of getting the values from the `global.css` to `tailwind.config.ts`

I think there can be some improvements on the DX with regards to the `global.css` since there is no easy way to reference the colors from VSCode even with color highlight extensions.

Reply:
Thanks for the tips.